try:
    # код, который может вызвать исключение
except Ошибка1:
    # обработка этой ошибки
except Ошибка2:
    # обработка другой ошибки
else:
    # выполняется, если не было ошибок
finally:
    # выполняется всегда